src/main/java/net/mcreator/ui/modgui/FoodGUI.java

```diff
@@ -96,7 +96,7 @@
 	private final DataListComboBox creativeTab = new DataListComboBox(mcreator);
 
 	private final Model normal = new Model.BuiltInModel("Normal");
-	private final SearchableComboBox<Model> renderType = new SearchableComboBox<>();
+	private final SearchableComboBox<Model> renderType = new SearchableComboBox<>(new Model[] { normal });
```

Comment:
How are these changes related to this PR?

Reply:
Tests fail without them

Reply:
https://github.com/MCreator/MCreator/blob/bf1d9772f9e839a6d5d8fc007d31befdc169ba91/src/main/java/net/mcreator/generator/Generator.java#L674-L675

Because of this line, I cannot pass null `GeneratableElement` here (or GE from `ModElement` as well), so I'm calling `getElementFromGui()`, which fails as soon as `requireNonNull()` throws an exception triggered by no elements present in model combo boxes.

Reply:
But there are cases where GeneratableElement can be null still, this is designed so

Reply:
With https://github.com/MCreator/MCreator/pull/2010/commits/15513a8afc2e8e386738954f919a2ff9870a32bd, I moved list templates groups creation to the constructor to 'relieve' the update thread a bit (and so that those tabs are always located next to each other). This means I need this element to not be null _at the very beginning_. What should I do?

Reply:
Templates can change groups so this is not ok. Also not sure how the commit https://github.com/MCreator/MCreator/commit/15513a8afc2e8e386738954f919a2ff9870a32bd is related to this PR. It should have own PR.

Reply:
>Also not sure how the commit `15513a8` is related to this PR

It's fourth commit in [this](https://github.com/MCreator/MCreator/pull/2010#commits-pushed-517c4be) list -> part of this PR.

Reply:
I know, but this change is not related to list templates

Reply:
>Oh, I get it!

That's about **the biggest part** of this commit, right? Then I know what to do.